### PR TITLE
Fake collector to provoke ACLK messages

### PR DESCRIPTION
**Pull request by amoss**
##### Summary

A fake collector that can be used to provoke the messages needed in the integration testing.

##### Component Name
ACLK

##### Test Plan

1. Install.
2. `aclk/tests/install-fake-charts.d.sh`  (may need to `chmod +x`)
3. Run the agent
4. Check the messages at the VerneMQ using the paho inspection and/or your own testing.

### CODE REVIEW

File: aclk/tests/fake_charts.d.plugin [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+echo "BEGIN aclk_test.newcol 1000000"
+echo "SET aclk1 = 3"
+echo "SET aclk2 = 3"
+echo "END"
+sleep 5
+echo "CHART aclk_test2.newcol '' 'Generate new collector/chart event' 'units' aclk_test aclk_test lines 900001 1"
+echo "DIMENSION aclk1 '' percentage-of-absolute 1 1"
+
+sleep 5
+exit 0   # Signal that we are done
```
Comment (ilyam8) [inline aclk/tests/fake_charts.d.plugin]: the plugin will be restarted by netdata if exit code is 0 if i remember it right, not sure it is important for the test case, just saying

Comment (amoss) [reply aclk/tests/fake_charts.d.plugin]: Might be the case - I remember seeing a message in the logs that said the collector had terminated and would not be restarted. If this is not the case during testing then flip the return code and see if we can provoke the collector deleted case.

File: aclk/tests/install-fake-charts.d.sh.in [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+TARGET="@pluginsdir_POST@"
+BASE="$(cd "$(dirname "$0")" && pwd)"
+
+cp "$BASE/fake-charts.d.plugin" "$TARGET/charts.d.plugin"
```
Comment (stelfrag) [inline aclk/tests/install-fake-charts.d.sh.in]: The file is `fake_charts.d.plugin` (not with a dash)

Comment (amoss) [reply aclk/tests/install-fake-charts.d.sh.in]: Moved it.

**Pull request merged**